internal/lsp/server_template_test.go

```diff
@@ -124,6 +116,73 @@ func TestTemplateContentsForFile(t *testing.T) {
 	}
 }
 
+func TestTemplateContentsForFileInWorkspaceRoot(t *testing.T) {
```

Comment:
Root is not allowed and is an error which can be silently ignored if the caller understands it.